### PR TITLE
RPC and validator healthcheck update 

### PR DESCRIPTION
**Pull request by baburgess**

### CODE REVIEW

File: packages/arb-node-core/nodehealth/nodehealth.go [hunk 1]
```diff
@@ -96,10 +99,24 @@ func updateConfig(config *configStruct, logMessage Log) {
 	defer config.mu.Unlock()
 	//Load the configuration message into the config struct
 	if logMessage.Var == "openethereumHealthcheckRPC" {
-		config.openethereumHealthcheckRPC = logMessage.ValStr
+		u, err := url.Parse(logMessage.ValStr)
+		if err != nil {
+			return
+		}
+		config.openethereumHealthcheckRPC = u.Hostname()
```
Comment (hkalodner) [inline packages/arb-node-core/nodehealth/nodehealth.go:106]: Not sure why we're stripping out the port here and delivering the hostname and port separately?

**Review by hkalodner — CHANGES_REQUESTED** [review]

File: packages/arb-node-core/nodehealth/nodehealth.go (hunk 1 above)
Comment (baburgess) [reply packages/arb-node-core/nodehealth/nodehealth.go:106]: The healthcheck is on a separate port from the RPC endpoint that is passed in via a command line argument.  This approach gives us configurability for the default healthcheck port inside the nodehealth file versus having to update it across the code base.

Comment (hkalodner) [reply packages/arb-node-core/nodehealth/nodehealth.go:106]: Ah cool

**Review by hkalodner — APPROVED** [review]
LGTM

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**